extensions/tower/src/commandOpenBookmarks.tsx

```diff
@@ -0,0 +1,140 @@
+import {
+  ActionPanel,
+  closeMainWindow,
+  Detail,
+  Icon,
+  List,
+  preferences,
+  render,
+  showHUD,
+  showToast,
+  ToastStyle,
+} from "@raycast/api";
+import { exec } from "child_process";
+import fs from "fs";
+import os from "os";
+import plist from "plist";
+import { promisify } from "util";
+import Bookmark from "./dtos/Bookmark.dto";
+import ImportedTowerBookmarks, { ImportedTowerBookmark } from "./interfaces/imported-tower-bookmark";
+
+const execp = promisify(exec);
+
+const towerBookmarksPlistLocation = "/Library/Application Support/com.fournova.Tower3/bookmarks-v2.plist";
+
+async function main() {
+  if (isTowerCliInstalled()) {
+    const bookmarks = await fetchBookmarks();
+
+    render(<BookmarkList bookmarks={bookmarks} />);
+  } else {
+    render(<Detail navigationTitle="Tower CLI not installed" markdown={towerCliRequiredMessage()}></Detail>);
+  }
+}
+
+main();
+
+async function fetchBookmarks(): Promise<Bookmark[]> {
+  try {
+    const bookmarksFile = os.homedir() + towerBookmarksPlistLocation;
+    const obj = plist.parse(fs.readFileSync(bookmarksFile, "utf8")) as unknown as ImportedTowerBookmarks;
+
+    if (obj.children.length === 0) {
+      await showToast(ToastStyle.Failure, "No Bookmarks found", "Now is the time to start bookmarking!");
+
+      return Promise.resolve([]);
+    }
+
+    const bookmarks = obj.children
+      .map((bookmark: ImportedTowerBookmark) => {
+        return new Bookmark(
+          bookmark.fileURL.replace("file:///", "/"),
```

Comment:
Check out [tildify](https://github.com/raycast/extensions/blob/c3426b01a2e2a6472fb6b4f5f1c8c21d90336e8d/extensions/visual-studio-code-recent-projects/src/index.tsx#L54) package for shortening all these `/Users/<nickname>/` paths.

Reply:
The input is like "file://Users.." so that why I added the replace. But added the tildify :) Thx!